components/StatusCheckBox/StatusCheckbox.tsx

```diff
@@ -0,0 +1,56 @@
+import { StatusCheckboxProps } from '@/utils/constants/leadslist-constant';
+import React from 'react';
+
```

Comment:
Change this file name to `index.tsx` and rename the folder name to `status-checkbox`